### ISSUE TITLE
rs train results

### ISSUE THREAD

**Issue by jaigsingla** [mapbox/robosat#211]
Hi ,

I have prepared tanzania dataset by downloading grid files and tiff files using routines given by rsp commands.  and now tiled dataset is available under correct directory structure ..

Now , while training with rsp i am facing issue of Unable to load data augmentation module 

so i am using rs train command . .... after 2nd epochs getting ... validation loss:0 , miou:nan, building IoU:nan, MCC:nan 

??request your comments on this.
